crates/ruff_linter/resources/test/fixtures/ruff/RUF031.py

```diff
@@ -0,0 +1,21 @@
+d = {(1,2):"a",(3,4):"b",(5,6,7):"c",(8,):"d"}
+d[(1,2)]
+d[(
+    1,
+    2
+)]
```

Comment:
Could we add one more example of the form

```python
token_features[
    (window_position, feature_name)
] = self._extract_raw_features_from_token
```

It's quiet common in the ecosystem check.